app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/renderBanner.ts

```diff
@@ -18,20 +19,20 @@ const hideQuestion = () => {
     callIfNotNull(question, element => removeElement(question))
 }
 
-const insertBanner = (userPreference) => {
+const insertBanner = (userPreference, preferenceCommunicator) => {
```

Comment:
We should move these function declarations inside renderBanner so we don't have to pass through parameters like this. insertBanner shouldn't really care about preferenceCommunicator.

Reply:
Yeah, I think the event handler should be configured elsewhere.